src/actions/index.js

```diff
@@ -0,0 +1,36 @@
+export const CLEAR = 'CLEAR'
+export const NUMBER_INPUT = 'NUMBER_INPUT'
+export const OPERATOR = 'OPERATOR'
+export const DECIMAL = 'DECIMAL'
+export const CALCULATE = 'CALCULATE'
+
+export const clearAction = () => {
+    return {
+        type: CLEAR
+    }
+}
+
+export const numberAction = (number) => {
+    return {
+        type: NUMBER_INPUT,
+        number
+    }
+}
```

Comment:
The format of action is: `{ type: ACTION_TYPE, payload: PARAMS }`

Reply:
Fixed, thank you for your review